src/scenes/level-selection-scene.ts

```diff
@@ -10,21 +10,19 @@ import { AudioPlayer } from "@components";
 import { getData, GameScore } from "@data";
 import { SelectedLevel } from "../Firebase/firebase-event-interface";
 import { FirebaseIntegration } from "../Firebase/firebase-integration";
-import {
-  createBackground,
-  levelSelectBgDrawing,
-} from "@compositions/background";
+import { createBackground, levelSelectBgDrawing } from '@compositions/background';
 import {
   PreviousPlayedLevel,
   LEVEL_SELECTION_BACKGROUND,
-  MAP_ICON_SPECIAL,
-  MAP_ICON,
-  MAP_LOCK,
-  STAR,
-  NEXT_BUTTON,
-  BACK_BUTTON,
-  AUDIO_INTRO,
-} from "../constants";
+  MAP_ICON_IMG,
+  MAP_ICON_SPECIAL_IMG,
+  MAP_LOCK_IMG,
+  STAR_IMG,
+  NEXT_BTN_IMG,
+  BACK_BTN_IMG,
+   AUDIO_INTRO,
+} from '@constants';
```

Comment:
lets use double quotes just to be consistent for imports

Reply:
Changes applied, thank you.